en/developer_guide/multiple_device.rst

```diff
@@ -0,0 +1,89 @@
+Multiple device
+===============
+
+Prerequisite
+------------
+
+To enable the multi-device rendering, we assume that the server in front of Php (Varnish, Apache2, Nginx, ...)
```

Comment:
"server in front of php" -> Are we speaking of the server running php, or a reverse proxy ? Or do you mean PHP-FPM is running behind a webserver ? The configuration is unclear to me.